docs/cloud-adoption/governance/governance-multiple-teams.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Governance design guide: new development in Azure for multiple teams"
+title: "Enterprise Cloud Adoption: Governance design in Azure for multiple teams"
 description: Guidance for configuring Azure governance controls for multiple teams, multiple workloads, and multiple environments
-author: petertay
+author: petertaylor9999
 ---
 
-# Governance design guide: new development in Azure for multiple teams
+# Enterprise Cloud Adoption: Governance design for multiple teams
 
-The goal of this guidance is to help you learn the process for designing a resource governance model in Azure.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
+The goal of this guidance is to help you learn the process for designing a resource governance model in Azure to support multiple teams, multiple workloads, and multiple environments.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
 
 Our requirements are:
 * Identity management for multiple teams with different resource access requirements in Azure. The identity management system stores the identity of the following users:
```

Comment:
Should Figure 1 show creating a subscription? Otherwise it doesn't seem to relate to the sentence just above. (I think you could delete that sentence and figure, because you already state the account owner can create/delete subscriptions.)

Reply:
> No custom RBAC roles

What motivates this requirement? The other requirements meet obvious organizational needs, but this one isn't clear to me.

Reply:
"Creating a subscription" is covered in the "Resource access management in Azure" doc. I'll link back to it.